Changes to link style

mcritz opened the pull request.

mcritz committed:
links have nicer hover styling, particularly headers

apontious committed:
Merge branch 'master' into design

NSCoderNight commented:
Underline and background on hover look great, but I really want the underline to be permanent, and only the background to be something that shows up on hover. Otherwise, I think it's too hard to pick out links from regular text given the dark purple we're using. I'm going to check in my changes that make that happen.

Only concern there is that the background color looks a _little_ odd on very large texts like the blog title links and the NSCoderNight text at the top of the page. But...I think it may still be for the best to make it consistent everywhere.

apontious committed:
Gemfile.lock change.

This is something that is changed automatically now in the master branch, and now here too since I merged master into this branch. *shrug*.

apontious committed:
Fiddle with links a bit more.

- Make underline permanent for all links but social-media-list links, where the icon + text just looks bad with an underline.
- Have background-on-hover behavior for all links, regardless of size.

apontious committed:
Another link experiment.

This time:
- Only links inside paragraphs of text have underline.
- All other links never have an underline, not even on hover.
- All links still have the purple background on hover.

apontious committed:
Merge branch 'master' into design

Conflicts:
	Gemfile.lock

NSCoderNight commented:
OK, my last two commits:

First: background on hover everywhere, underline everywhere (except the social media entries at the bottom)
Second: background on hover everywhere, underline only for links in paragraphs.

I think I like the second one better, but not sure.

NSCoderNight commented:
Enh, let's go with it for now!
